### PR TITLE
[#7970] Platform: Added Backup Type label to distinguish between YSQL and YCQL Backups.

### PR DESCRIPTION
**Pull request by gaurav061**
Description: There is no distinguishing parameter for YSQl and YCQL Schedule Backups. Added a new label to display backup type i.e YCQL and YSQL for their corresponding type. for another backup type, this field will show an empty string i.e Yedis.

Test Plan: 
1. Created a YSQL type backup and the same backup type was shown in the label.
2. Created a YCQL type backup and the same backup type was shown in the label.

### CODE REVIEW

**Comment by gaurav061** [comment]
**Before** 
Before_change

**After**
After_change
Sample screenshot.

**Review by jaydeepkumara — APPROVED** [review]

File: managed/ui/src/components/schedules/Schedules.js [hunk 1]
```diff
@@ -75,6 +75,19 @@ class ScheduleDisplayItem extends Component {
       !schedule.taskParams.timeBeforeDelete || schedule.taskParams.timeBeforeDelete === 0
         ? 'Unlimited'
         : moment.duration(schedule.taskParams.timeBeforeDelete).humanize();
+    
+    let backupType = '';
```
Comment (nishantSharma459) [inline managed/ui/src/components/schedules/Schedules.js:79]: Can we declare all the variables on the top?

File: managed/ui/src/components/schedules/Schedules.js [hunk 2]
```diff
@@ -106,6 +119,9 @@ class ScheduleDisplayItem extends Component {
             <DescriptionItem title={'Schedule (UTC)'}>
               <span>{schedule.frequency || schedule.cronExpression}</span>
             </DescriptionItem>
+            <DescriptionItem title={'Backup Type'}>
```
Comment (nishantSharma459) [inline managed/ui/src/components/schedules/Schedules.js:122]: `title='Backup Type'`. No need for curly braces.

File: managed/ui/src/components/schedules/Schedules.js [hunk 3]
```diff
@@ -75,6 +75,19 @@ class ScheduleDisplayItem extends Component {
       !schedule.taskParams.timeBeforeDelete || schedule.taskParams.timeBeforeDelete === 0
         ? 'Unlimited'
         : moment.duration(schedule.taskParams.timeBeforeDelete).humanize();
+    
+    let backupType = '';
+    switch (schedule?.taskParams?.backupType) {
```
Comment (nishantSharma459) [inline managed/ui/src/components/schedules/Schedules.js:80]: We can destructure it. Initialize a variable and then put it inside the switch.

**Review by nishantSharma459 — COMMENTED** [review]
Please address the comments.

File: managed/ui/src/components/schedules/Schedules.js (hunk 3 above)
Comment (gaurav061) [reply managed/ui/src/components/schedules/Schedules.js:80]: The structure is not deeply nested so, I am leaving it for now

File: managed/ui/src/components/schedules/Schedules.js (hunk 2 above)
Comment (gaurav061) [reply managed/ui/src/components/schedules/Schedules.js:122]: Done

File: managed/ui/src/components/schedules/Schedules.js (hunk 1 above)
Comment (gaurav061) [reply managed/ui/src/components/schedules/Schedules.js:79]: Done

**Review by nishantSharma459 — APPROVED** [review]

**Review by sshev — APPROVED** [review]

**Pull request merged**